test/unit/template_importer_test.rb

```diff
@@ -225,18 +225,57 @@ def self.import!(name, text, _metadata)
       end
     end
 
+    test 'should import files from git' do
+      setup_settings
+      imp = ForemanTemplates::TemplateImporter.new
```

Comment:
It should be ok now, all I want to test here is that the right type of import source is chosen when repo is specified with https://. `import_from_git` is stubbed in test, so there is no calling out to external repos.